src/test/java/lotto/domain/MoneyTest.java

```diff
@@ -21,6 +21,8 @@ void NotEnoughMoneyTest() {
     void getTest() {
         Money money = new Money(10000);
 
-        assertThat(money.get()).isEqualTo(10000);
+        assertThat(money.toDTO()
+                .get())
+                .isEqualTo(10000);
```

Comment:
```suggestion
        assertThat(money.toDTO().get()).isEqualTo(10000);
```
함수의 파라미터로 들어가있는 경우는 제외하고 카운트 하신다고 생각하시면 돼요.